### DISCUSSION TITLE
How to disconnect from a hung up SSH connection?

### DISCUSSION THREAD

**Discussion by huylv** [wezterm/wezterm#5317] [Q&A] [answered]
### What Operating System(s) are you running on?

Linux X11

### Which Wayland compositor or X11 Window manager(s) are you using?

Gnome 3

### WezTerm version

20240203-110809-5046fc22

### Ask your question!

When there is a network issue, my SSH connection hangs up. I want to be able to reset the current pane and disconnect from the SSH server. Is it possible to do so in wezterm? In iTerm2, one can easily do that by resetting the pane via right-click menu.

**Comment by bew** [comment, marked as answer]
How are you ssh-ing? Through `wezterm ssh`? Or using the usual `ssh` command?

Maybe this can help:
https://superuser.com/questions/467398/how-do-i-exit-an-ssh-connection#620787
Aka `Enter` `~` `.`

**Reply by huylv** [reply]
Oh wow, thank you so much. I didn't know this before. I normally use just the usual `ssh`.
